packages/office-ui-fabric-react/src/utilities/color/interfaces.ts

```diff
@@ -6,7 +6,9 @@ export interface IRGB {
   g: number;
   /** Blue, range 0-255. */
   b: number;
-  /** Alpha, range 0 (transparent)-100. Usually assumed to be 100 if not specified. */
+  /** Alpha, range 0 (transparent)-100. Usually assumed to be 100 if not specified.
+   * When useTransparencySlider is true, alpha value represents the transparentness
```

Comment:
Thinking about this more, it could confuse consumers to have a value "alpha" which is sometimes transparency. Also, none of the color conversion functions will handle that correctly.

Instead, I think it would be better to store the value as actual alpha, and add a public conversion function (in this folder) for getting transparency from a color. (The color slider can convert the alpha value to transparency only when displaying it.) Sorry for not providing this feedback sooner.

Reply:
So i have removed this comment, i misinterpreted your earlier comment regarding putting a comment in the IColor Interface. So at all times the value is alpha only, we are just showing transparency on UI end, but at all times the "alpha" variable actually holds the alpha value only and not the transparency.